File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    line = s.split(" ")
    for word in line:
        for i in range(len(word)):
            if i % 2 == 0:
                answer += word[i].upper()
            else:
                answer += word[i].lower()
        answer += " "
    return answer[:-1]

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] line = s.split(" ", -1);
        int N = line.length;
        for (int i = 0; i < N; i++) {
            for (int j = 0; j < line[i].length(); j++) {
                if (j % 2 == 0) {
                    answer += Character.toUpperCase(line[i].charAt(j));
                } else {
                    answer += Character.toLowerCase(line[i].charAt(j));
                }
            }
            if (i < N-1) {
                answer += " ";
            }
        }
        return answer;
    }
}